extensions/annotations/src/main/java/io/opentelemetry/extension/annotations/SpanAttribute.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.extension.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation marks that a parameter of a method annotated by the {@link WithSpan} annotation
+ * should be added as an attribute to the newly created {@link io.opentelemetry.api.trace.Span}.
+ * Using this annotation is equivalent to calling {@code Span.currentSpan().setAttribute(...)}
+ * within the body of the method.
+ *
+ * <p>Application developers can use this annotation to signal OpenTelemetry auto-instrumentation
```

Comment:
This seems a copy-paste :) Probably should read: 
```
* <p>Application developers can use this annotation to signal OpenTelemetry auto-instrumentation
*  that a new span attribute should be added to a span created when the parent method is executed.
```

Reply:
Good catch, indeed a copypasta fail!